R/configure_forecast_run.R

```diff
@@ -49,9 +49,7 @@ get_lag_periods <- function(lag_periods,
                     "quarter" = c(1,2,3,4),
                     "month" =  c(1, 2, 3, 6, 9, 12),
                     "week" = c(1, 2, 3, 4, 8, 12, 24, 48, 52),
-                    "day" = c(1, 2, 3, 4, 5, 6, 7, 14, 
-                              21, 28, 28*2, 28*3, 28*6, 
-                              28*9, 28*12, 365)
+                    "day" = c(7, 14, 21, 28, 60, 90, 180, 365)
```

Comment:
I am in general concerned about #77 - Do we want to address this as a part of this PR ?

Reply:
Great call out. My comments are in the issue #77

Reply:
I propose tackling this in another PR